content/docs/conditional-rendering.md

```diff
@@ -116,13 +117,14 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**Experimente no CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
 
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+Declarar uma variável e usar uma declaração condicional `if` é uma ótima maneira de renderizar um componente, mas 
+às vezes você pode querer usar uma sintaxe mais curta. Existem algumas maneiras para utilizar condições inline em JSX, explicadas abaixo.
```

Comment:
I think this ENTER does not exist, may break formatting on site.

Reply:
Agreed.